src/Microsoft.ML.Data/Transforms/ExtensionsCatalog.cs

```diff
@@ -70,11 +70,27 @@ public static ColumnSelectingEstimator DropColumns(this TransformsCatalog catalo
             => ColumnSelectingEstimator.DropColumns(CatalogUtils.GetEnvironment(catalog), columnsToDrop);
 
         /// <summary>
-        /// ColumnSelectingEstimator is used to select a list of columns that user wants to drop from a given input.
+        /// ColumnSelectingEstimator is used to select a list of columns that user wants to keep from a given input.
         /// </summary>
+        /// <remarks>
+        /// <format type="text/markdown">
+        /// <see cref="SelectColumns"/> operates on the schema of an input IDataView,
+        /// either dropping unselected columns from the schema or keeping them but hiding them from the user. Keeping columns hidden
```

Comment:
> keeping them but hiding them from the user [](start = 65, length = 43)

What do you mean by keeping them, but hiding them from the user? This meaning might get a bit confusing with KeepingColumns hidden.